Fix products import issue with categories. Default categories paths are taken from store 1, this patch sets store to 0.
This fix is needed to import products on UEFA.

diff --git a/Model/Import/Product/CategoryProcessor.php b/Model/Import/Product/CategoryProcessor.php
index 54fdecb..be80c40 100644
--- a/Model/Import/Product/CategoryProcessor.php
+++ b/Model/Import/Product/CategoryProcessor.php
@@ -74,7 +74,8 @@ class CategoryProcessor
             $collection = $this->categoryColFactory->create();
             $collection->addAttributeToSelect('name')
                 ->addAttributeToSelect('url_key')
-                ->addAttributeToSelect('url_path');
+                ->addAttributeToSelect('url_path')
+                ->setStoreId(\Magento\Store\Model\Store::DEFAULT_STORE_ID);
             /* @var $collection \Magento\Catalog\Model\ResourceModel\Category\Collection */
             foreach ($collection as $category) {
                 $structure = explode(self::DELIMITER_CATEGORY, $category->getPath());
